packages/e2e-tests/specs/editor/blocks/list.test.js

```diff
@@ -539,4 +539,26 @@ describe( 'List', () => {
 		);
 		expect( content ).toMatchSnapshot();
 	} );
+
+	// Regression test for https://github.com/WordPress/gutenberg/issues/33078
+	it( 'pasting from paragraph blocks work', async () => {
+		const text = 'Copy and paste me, pretty please!';
+
+		await insertBlock( 'Paragraph' );
+		await page.keyboard.type( text );
+		await pressKeyWithModifier( 'primary', 'a' );
+		await pressKeyWithModifier( 'primary', 'c' );
+		await page.keyboard.press( 'ArrowDown' );
+
+		await page.keyboard.press( 'Enter' );
+		await insertBlock( 'List' );
+		await pressKeyWithModifier( 'primary', 'v' );
+
+		const content = await page.$eval(
+			'.wp-block-list',
+			( el ) => el.innerHTML
+		);
```

Comment:
We tend to use `getEditedPostContent()` from `@wordpress/e2e-test-utils` for cases like these, I think. (Which would mean comparing to a whole snapshot, e.g. via `expect( content ).toMatchInlineSnapshot( ... )`.

If that seems to heavy-handed, maybe we can at least change the selector here: Overall, the rule of thumb is to use something "user-facing" in e2e tests, rather than a class name (like `.wp-block-list`), as the latter is considered an implementation detail (that might change over time).

Often, we tend to use `aria-label`s as selectors, e.g. 

```suggestion
		const content = await page.$eval(
			'[aria-label="Block: List"]',
			( el ) => el.innerHTML
		);
```